hardware/computemodule/cm-emmc-flashing.md

```diff
@@ -81,24 +93,23 @@ Now build and install the `usbboot` tool:
 
 ```bash
 make
-sudo make install
 ```
 
 Run the `usbboot` tool and it will wait for a connection:
 
 ```bash
-sudo rpiboot
+sudo .\rpiboot
```

Comment:
Forward slash, surely. And don't we want to install rpiboot (see the deleted make install above)?